api/http/api_devauth_test.go

```diff
@@ -1423,18 +1423,11 @@ func TestApiV2DevAuthDeleteDeviceAuthSet(t *testing.T) {
 			code: http.StatusNoContent,
 			err:  nil,
 		},
-		{
-			req: test.MakeSimpleRequest("DELETE",
-				"http://1.2.3.4/api/management/v2/devauth/devices/foo/auth/bar", nil),
-			code: http.StatusInternalServerError,
-			body: RestError("internal error"),
-			err:  store.ErrAuthSetNotFound,
-		},
```

Comment:
next test is checking this; also I've changed the behavior here so if the user will try to remove unexisting authentication set he will get 404 instead of 500

Reply:
ok thanks.